src/app/cases/page.tsx

```diff
@@ -94,6 +100,26 @@ export default function Page() {
               setCaseFilters({ ...caseFilters, countries: v as string[] })
             }
           />
+          <AuthButtons>
+            {profile && profile.profileData ? (
```

Comment:
can we check for authentication here, as opposed to onboarded-ness? if the user is logged in but not yet onboarded, we don't want to prompt them sign up/log in instead.

this does mean that if the user is authenticated & not onboarded, we would see the profile/settings button up here. @davidqing6432 can we make a special settings page for un-onboarded users? have added this to https://linear.app/cal-blueprint/issue/IJP-84/refinements-home-page

Reply:
yeah this is just a fill-in because the updated profile context (in onboarding context sprint) is not here. should i copy paste it and check for user id or should i wait for the auth context?

Reply:
given timelines, i feel like auth context may need to be pushed to spring since it's technically not _essential_ to mtp! so let's have this work without the auth context for rn

but yes i think onboarding context is ready to go, so we can merge that in, rebase this off of that, and then use profile context here?

Reply:
sounds good, i can rebase